src/components/form/form.stories.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import {
+  select, boolean, number, text
+} from '@storybook/addon-knobs';
+
+import { dlsThemeSelector } from '../../../.storybook/theme-selectors';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import Form from './form.component';
+import Textbox from '../../__experimental__/components/textbox';
+import Button from '../button';
+import DialogFullScreen from '../dialog-full-screen';
+import Dialog from '../dialog';
+
+export default {
+  title: 'Test/Form',
+  component: Form,
+  parameters: {
+    themeSelector: dlsThemeSelector,
+    info: {
+      disable: true
+    },
+    knobs: { escapeHTML: false }
+  }
+};
+
+export const Basic = () => {
+  const { defaultProps } = Form;
+
+  const buttonAlignment = select(
+    'buttonAlignment',
+    OptionsHelper.alignBinary,
+    defaultProps.buttonAlignment
+  );
+
+  const showSummary = boolean('showSummary', defaultProps.showSummary);
```

Comment:
As we are hoping to streamline our regression testing (see link below), it might be worth moving away from using `addon-knobs`, as it adds a lot of overhead, in favour of adding multiple demos to display the various configurations of the new Form.

https://github.com/Sage/carbon/diffs/1?base_sha=5dcd7ef085344df9c974a4232859cc4fbc4fe165&commentable=true&pull_number=2774&sha1=5dcd7ef085344df9c974a4232859cc4fbc4fe165&sha2=95389c39933f7e1723181501d5c3d621155b8ca1&short_path=2c80563&unchanged=expanded#motivation

Reply:
@edleeks87 I've addressed this comment and removed `form.stories.js` - everything is in `form.stories.mdx` now